debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
Do not do this, instead, install ../misc/mgmt.service directly within mgmt.install